transitions/registration.js

```diff
@@ -27,6 +27,26 @@ var getRegistrations = function(db, patientId, callback) {
     utils.getRegistrations({ db: db, id: patientId }, callback);
 };
 
+
+var addValidationErrors = function(registrationConfig, doc, errors) {
```

Comment:
Nothing changed, I just extracted it out to make the place where it's called read cleaner.